packages/sdk/src/constant.ts

```diff
@@ -30,3 +30,5 @@ export const MFModuleType = {
   NPM: 'npm',
   APP: 'app',
 };
+
+export const MODULE_DEVTOOL_IDENTIFIER = 'garfishModuleDebug';
```

Comment:
If this subsequent consideration maintains two copies you can change the value here